raw_data/preprocessed/africa/uganda_english.txt

```diff
@@ -0,0 +1,364 @@
+
+PART V - ENVIRONMENTAL REGULATION
+
+Environmental impact assessment. 
+(20) [1] A developer of a project described in the Third Schedule to this Statute Environmental shall submit a project brief to the lead agency, in the prescribed form and giving the prescribed information. [2] The Minister may, on the advice of the Board, by statutory instrument, amend the Third Schedule. [3] An environmental impact assessment shall be undertaken by the developer where the lead agency, in consultation with the Executive Director, is of the view that the project: [a] may have an impact on the environment; [b] is likely to have a significant impact on the environment; or [c] will have a significant impact on the environment. [4] An environmental impact assessment shall be undertaken by experts whose names and qualifications are approved by the Authority. [5] An environmental impact assessment required in subsection [3] shall be appropriate to the scale and possible effects of the project, and accordingly: [a] where the project may have an impact on the environment, an environmental impact review shall be conducted; [b] where the project is likely to have an impact on the environment, an environmental impact evaluation shall be conducted; or [c] where the project will have a significant impact on the environment, an environmental impact study shall be conducted. [6] Where the lead agency, in consultation with the Authority, is satisfied that an environmental impact review or an environmental impact evaluation conducted in accordance with paragraphs [a] or [b] of subsection [5] does not disclose possible significant impact on the environment, it may approve the environmental aspects of the project. [7] Where the lead agency, in consultation with the Authority is satisfied, after considering the environmental impact review or the environmental impact evaluation, that the project will lead to significant impact on the environment, it shall require that an environmental impact study be conducted. [8] The Authority shall, in consultation with a lead agency, adopt guidelines with respect to environmental impact reviews and environmental impact evaluations, and environmental impact study on: [a] their format and contents; impact assessment [b] the procedure for conducting the assessments; [c] the participation of the public, especially those most affected by the project in the assessment; [d] any other matter the Authority considers relevant. [9] The conduct of an environmental impact assessment under this section shall be publicized in such a manner as may be prescribed.
+
+Environmental impact statement.
```

Comment:
I'm going to conclude the review here. As my comments illustrate, there is text from the margin of the original legislation in the provisions in this preprocessed legislation. Please re-read this preprocessed legislation, ensure that your script removes all such text, and post a new revision.